### PR TITLE
[TD-13342]<fix>: fix CI test failed when taosadapter.toml is missing

### PR DESCRIPTION
**Pull request by zyyang90**
[TD-13342]<fix>: fix CI test failed when taosadapter.toml is missing

### CODE REVIEW

File: tests/develop-test/3-connectors/java/test.sh [hunk 1]
```diff
@@ -14,6 +15,9 @@ stopTaosd
 rm -rf /var/lib/taos/*
 rm -rf /var/log/taos/*
 nohup taosd -c /etc/taos/ > /dev/null 2>&1 &
+if [ ! -f /etc/taos/taosadapter.toml ]; then
+  cp ../../../../src/plugins/taosadapter/example/config/taosadapter.toml /etc/taos/taosadapter.toml
```
Comment (gccgdb1234) [inline tests/develop-test/3-connectors/java/test.sh]: 这里应该利用前面设置好的  WKC 代替 ../../../.. 这样极其容易出错的写法

Comment (gccgdb1234) [reply tests/develop-test/3-connectors/java/test.sh]: 描述不太准确， WKC是在后面设置的，那么应该挪到前面去，在这里也可以复用。

**Review by sangshuduo — COMMENTED** [review]
I would recommend launching `taosadapter` from the specified location and specified `taosadapter.toml` with -c instead of copy it to the system directory to avoid infecting system or other app/test needs to write it too.

**Comment by zyyang-taosdata** [comment]
> I would recommend launching `taosadapter` from the specified location and specified `taosadapter.toml` with -c instead of copy it to the system directory to avoid infecting system or other app/test needs to write it too.

solved

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**